exercises/practice/leap/.approaches/introduction.md

```diff
@@ -0,0 +1,73 @@
+# Introduction
+
+There are various idiomatic approaches to solve Leap.
+You can use a chain of boolean expressions to test the conditions.
+Or you can use a [ternary operator][ternary-operator].
+Another approach you can use is a [switch][switch] statement.
+
+## General guidance
+
+The key to solving Leap is to know if the year is evenly divisible by `4`, `100` and `400`.
+For determining that, you will use the [remainder operator][remainder-operator].
+
+## Approach: Chain of Boolean expressions
+
+```javascript
+export function isLeap(year) {
+  return year % 4 == 0 && (year % 100 != 0 || year % 400 == 0);
+}
+```
+
+For more information, check the [Boolean chain approach][approach-boolean-chain].
```

Comment:
Feel free to ignore this, but I've added a one-line description of the approach before linking to the approach. See https://exercism.org/tracks/csharp/exercises/collatz-conjecture/dig_deeper for an example. If you think that isn't really that useful, feel free to ignore.

Reply:
Yeah, I can see that, it was indeed for the student's benefit.